OpenDreamShared/Compiler/CompilerError.cs

```diff
@@ -45,6 +45,7 @@ public enum WarningCode {
         PointlessBuiltinCall = 2206, // For pointless calls to issaved() or initial()
         SuspiciousMatrixCall = 2207, // Calling matrix() with seemingly the wrong arguments
         FallbackBuiltinArgument = 2208, // A builtin (sin(), cos(), etc) with an invalid/fallback argument
+        SuspiciousAreaContentsEnumeration = 2209, // See https://github.com/OpenDreamProject/OpenDream/issues/1003
```

Comment:
3000-3999 may be a better place to put this, since this affects no compile-time behavior.